sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -99,6 +99,38 @@ class AdminOperations {
         
     }
     
+    static func loadSchool(id: Int, completion: ((School) -> Void), failure: (String) -> Void) {
+        let requestURL = StringConstants.kEndpointBaseURL + "/schools/" + String(id)
```

Comment:
you can make a method in StringConstants where you pass in the id and itll return the correct url. so its like StringConstants.kDetailSchoolURL(id) or osmething like that